main.nf

```diff
@@ -613,32 +613,129 @@ process dia_spectral_library_search {
         -mz_extraction_window_unit ${params.mz_extraction_window_unit} \\
         -mz_extraction_window_ms1_unit ${params.mz_extraction_window_ms1_unit} \\
         -rt_extraction_window ${params.rt_extraction_window} \\
-        -min_upper_edge_dist ${params.min_upper_edge_dist} \\
+	-extra_rt_extraction_window ${params.extra_rt_extraction_window} \\
+	-ion_mobility_window ${params.ion_mobility_window} \\
+	-im_extraction_window_ms1 ${params.im_extraction_window_ms1} \\
+	-use_ms1_ion_mobility ${params.use_ms1_ion_mobility} \\
+	-matching_window_only ${params.matching_window_only} \\
+	-irt_mz_extraction_window ${params.irt_mz_extratcion_window} \\
+	-irt_mz_extraction_window_unit ${params.irt_mz_extraction_window_unit} \\
+       -irt_im_extraction_window ${params.irt_im_extraction_window} \\
+	-split_file_input ${params.split_file_input} \\
+	-use_elution_model_score ${params.use_elution_model_score} \\
+	-mz_correction_function ${params.mz_correction_function} \\
+	-extraction_function ${params.extraction_function} \\
+	-outer_loop_threads ${params.outer_loop_threads} \\
+	-ms1_isotopes ${params.ms1_isotopes} \\
+	-no_progress ${params.no_progress} \\
+	-Calibration:ms1_im_calibration ${params.ms1_im_calibration} \\
+	-Calibration:im_correction_function ${params.im_correction_function} \\
+	-Calibration:debug_im_file ${params.debug_im_file} \\
+	-Calibration:debug_mz_file ${params.debug_mz_file} \\
+       -Library:retentionTimeInterpretation ${params.retentionTimeInterpretation} \\
+	-Library:override_group_label_check ${params.override_group_label_check} \\
+	-Library:force_invalid_mods ${params.force_invalid_mods} \\
+	-min_upper_edge_dist ${params.min_upper_edge_dist} \\
+	-sonar ${params.sonar} \\
         -RTNormalization:alignmentMethod ${params.irt_alignment_method} \\
         -RTNormalization:estimateBestPeptides \\
         -RTNormalization:outlierMethod none \\
+	-RTNormalization:useIterativeChauvenet ${params.useIterativeChauvenet} \\
+	-RTNormalization:RANSACMaxIterations ${params.ransacMaxIterations} \\
+        -RTNormalization:RANSACMaxPercentRTThreshold ${params.ransacMaxPercentRTThreshold} \\
```

Comment:
It seems to me the indentation on the left side is not right ( different numbers of white spaces)